Vagrantfile

```diff
@@ -165,22 +165,13 @@ vvv_config['hosts'] = [] unless vvv_config['hosts'].is_a? Hash
 vvv_config['hosts'] += ['vvv.test']
 
 vvv_config['sites'].each do |site, args|
-  if args.is_a? String
```

Comment:
I don't think we should remove the defaults

Reply:
I left only what was needed on the ruby side of things.

Reply:
One of the things I was hoping for was to JSON encode the final config option as Vagrant saw it and write it out, either for debugging purposes or just to act as a source of ground truth. I don't mind provisioners checking for this stuff too, that just adds resilience

Reply:
Could we leave this to another PR? We could improve the yml parsing logic in bash to print this file, but since it doesn't exist yet, could we do this separately?

Technically this debug data won't be tied to Vagrant/Ruby, so we need to do it on the VM/Container itself.

Reply:
Oh I agree JSON encoding should be another PR, but it wouldn't be possible with these bits removed. We should keep the vagrant based defaults. The bash based stuff should add to it, rather than replace it